examples/react/large-file-based/README.md

```diff
@@ -0,0 +1,6 @@
+# Example
```

Comment:
add explanation what this example does
explain how routes are generated using `pnpm run ...`

Reply:
Enough for you? :)